test/e2e/flask/solana/assets-list.spec.ts

```diff
@@ -0,0 +1,49 @@
+import { Suite } from 'mocha';
+import AssetListPage from '../../page-objects/pages/home/asset-list';
+import HeaderNavbar from '../../page-objects/pages/header-navbar';
+import SelectNetwork from '../../page-objects/pages/dialog/select-network';
+import { withSolanaAccountSnap } from './common-solana';
+
+describe('Multichain Asset List', function (this: Suite) {
+  const NETWORK_NAME_MAINNET = 'Ethereum Mainnet';
+  const LINEA_NAME_MAINNET = 'Linea Mainnet';
+  const SOLANA_NAME_MAINNET = 'Solana Mainnet';
+
+  it('displays all assets in the asset list for the selected network', async function () {
+    await withSolanaAccountSnap(
+      {
+        title: this.test?.fullTitle(),
+        showNativeTokenAsMainBalance: false,
+        mockCalls: true,
+      },
+      async (driver) => {
+        const assetListPage = new AssetListPage(driver);
+        const headerNavbar = new HeaderNavbar(driver);
+        const selectNetworkDialog = new SelectNetwork(driver);
+        await assetListPage.check_tokenItemNumber(3);
+        await assetListPage.check_tokenExistsInList('Solana', '50');
+        await assetListPage.check_tokenExistsInList('PUMPKIN', '6');
```

Comment:
Not a blocker, but what about creating variables with the token names? so we do not duplicate them?